catalog/transformer/catalog-transformer-tika-input/src/test/java/ddf/catalog/transformer/input/tika/TikaInputTransformerTest.java

```diff
@@ -779,4 +799,48 @@ private String convertDate(Date date) {
   private Metacard transform(InputStream stream) throws Exception {
     return tikaInputTransformer.transform(stream);
   }
+
+  private void addMetadataExtractor(MetadataExtractor metadataExtractor) {
+    ServiceReference<MetadataExtractor> serviceReference = mock(ServiceReference.class);
+    when(bundleCtx.getService(serviceReference)).thenReturn(metadataExtractor);
+    tikaInputTransformer.addMetadataExtractor(serviceReference);
+  }
+
+  private class TestMetacardExtractor implements MetadataExtractor {
```

Comment:
Do we need this whole class or could we just mock something out.

Reply:
+1